.github/linters/.rubocop.yml

```diff
@@ -0,0 +1,6 @@
+AllCops:
+  DisabledByDefault: true
+  TargetRubyVersion: 3.3.0
+
+Style/RedundantReturn:
```

Comment:
Is this rule really needed? I am not sure the changes from the rule make the code more readable.

Reply:
I started this PR thinking that we could use some rules from Rubocop to help organize and standardize some of the Ruby codebase.  

I think it is better to be consistent and removing these redundant returns sets a code standard going forwards. These extra returns are really extra characters and data that is not needed.

Maybe this rule is not needed but I think there could be some other Rubocop rules that are useful for mruby.